app.js

```diff
@@ -113,17 +110,23 @@ app.use(function(req, res, next) {
   next(globalError(404))
 })
 
-// handle global errors.
-app.use(function(err, req, res) {
-  res.locals.message = err.message
-  res.locals.error = req.app.get('env') === 'development' ? err : {}
+// Pass error information to res.locals
+app.use((err, req, res, next) => {
+  let errObj = {}
+
+  let status = err.status || err.statusCode || 500
+  status = status < 400 ? 500 : status
```

Comment:
Wouldn't this set some of our 200, 300 status codes to 500? 🤔

Reply:
Okay! Fair enough: https://github.com/cds-snc/cra-claim-tax-benefits/pull/254/commits/75c1b23ef865942ed6c4c3b2bf4a09a0cb850320

Reply:
I asked myself the same question but now I understand the behavior. Looks good to me then.